build.gradle

```diff
@@ -82,20 +122,18 @@ task initSubmodules {
     dependsOn updateSubmodules
   }
 }
-test.dependsOn initSubmodules
 
-task testPython(type:Exec) {
-  testPython() {
-      group = "verification"
-      description = "Runs the Python unit tests."
+task remindSubmodules {
+  onlyIf {
+    testJava.state.failure != null || testPython.state.failure != null
+  }
+  doLast {
+    logger.warn('Failed tests detected. Consider checking submodule status and updating with \'./gradlew updateSubmodules\' where necessary.')
```

Comment:
```suggestion
    logger.warn('Failed tests detected. Some tests depend on DCPs from the git submodule in test/RapidWrightDCP, consider checking status and updating with 'gradlew updateSubmodules'')
```

Reply:
The `./` doesn't work in Windows, so perhaps we remove it?  I also have been using it, but it appears that after a `git clone`, the `gradlew` script has the executable attribute in Linux.  So, perhaps its not needed in Linux either.